packages/visual-editor/src/components/Locator.tsx

```diff
@@ -1095,8 +1095,14 @@ const LocatorInternal = ({
     );
   }, [searchFilters]);
 
+  const facetsInState = useSearchState((state) => state.filters.facets);
```

Comment:
Can we consolidate this logic?
```suggestion
  const facetsWithOptions = 
    useSearchState(
      (state) => state.filters.facets?.filter((f) => f.options.length));
```
or even
```suggestion
  const hasFacetOptions = 
    useSearchState(
      (state) => state.filters.facets?.filter((f) => f.options.length))?.length > 0 ?? false;
```